### DISCUSSION TITLE
[广告]狡兔屋开始招人啦！

### DISCUSSION THREAD

**Discussion by RETRN1** [share121/inter-knot#1404] [其他]
![屏幕截图 2024-08-11 162643](https://github.com/user-attachments/assets/17a5ff12-4a62-46bc-a55f-433643f7d2ce)
狡兔屋开始招人啦！目前已有4名成员。加入我们，包吃包住！定期发工钱！每天都有好吃的青花鱼哦！
![215fb5090ba85e039ddd87fac2aae22d_8425932520024716177](https://github.com/user-attachments/assets/e0d66785-f897-4215-a72a-ffe226aac733)
![fba60dd20904524c0927f5992928c1e7_8086207329439057590](https://github.com/user-attachments/assets/8df4930d-766f-44a2-976e-b9c2986bd90f)
![af8f2b802a4e7b6b1b9ecb1201f7f11c_6049355643711622422](https://github.com/user-attachments/assets/b6ff5ba6-e34b-46e7-ac9d-e482d7d11e8f)
![13a099a6bbe7ded8dbbce4b3c78f8e86_4318385003976319491](https://github.com/user-attachments/assets/05b3ee9d-c485-4048-b61a-15e435d2387f)

**Comment by AnbulesPeo** [comment]
狡兔屋？那个差评榜上面排到前面的那个吗？

**Comment by gaojiaobei** [comment]
111，加入狡兔屋可以让妮可老大还清债款吗

**Comment by WORLD-executeME** [comment]
狡兔屋还是先把财报上的赤字解决再开始找人吧...

**Comment by Yukio4869** [comment]
听说狡兔屋经常拖欠员工工资，是真的吗？

**Comment by uidsd604** [comment]
我昨天在六分街碰到他们老大了，挺大个人了，去拉面馆领免费的东西都要变装好几次去领好几份，这个组织真的靠谱吗

**Comment by Jiucheng258** [comment]
真的吗？进你们狡兔屋后可以抢安比的汉堡吃吗？233，安比真的好可爱啊。

**Comment by Inter-Knot-User** [comment]
### 避雷！！这组织纯诈骗，老板拖欠工资暂且不提，还到处作恶！

**Comment by GenshinOnly** [comment]
111，工资无所谓，我要和安比拍拖

**Comment by uyfh352** [comment]
老板付不起工資可以肉偿嗎

**Comment by SSTV-10068** [comment]
您好，您的委托可以挂在SSTV上的广告位上喔~
SSTV广告位价格实惠，广告醒目吸人，[欢迎大家来抢购！](https://github.com/share121/inter-knot/discussions/1065)

**Comment by laotiehaha** [comment]
下属能和上司啵嘴吗？

**Reply by RETRN1** [reply]
应该能

**Comment by RETRN1** [comment]
我只是给那个粉头发的女子打广告，她说会给我工钱

**Comment by LYBQ** [comment]
狡兔屋！！！还钱

**Comment by XIong2210155** [comment]
安可在床上好乖
![Uploading 0238CEAF.png…]()

、

**Comment by aizai1173** [comment]
请问加入狡兔屋可以包分配嘛？！！
